### PR TITLE
Separate Key Vault mgmt plane tests from data plane

### PR DESCRIPTION
**Pull request by heaths**
Relates to #37421

### CODE REVIEW

**Review by m-nash — APPROVED** [review]

File: sdk/keyvault/tests.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ extends:
   template: ../../eng/pipelines/templates/stages/archetype-sdk-tests.yml
   parameters:
     ServiceDirectory: keyvault
+    SDKType: client
```
Comment (pallavit) [inline sdk/keyvault/tests.yml:7]: What SDKType do we use for mgmt?

Comment (heaths) [reply sdk/keyvault/tests.yml:7]: "mgmt"

Comment (heaths) [reply sdk/keyvault/tests.yml:7]: @m-nash do you want me to add live tests for mgmt e.g., a _tests.mgmt.yml_? Doesn't seem typical - just _ci.mgmt.yml_. /cc @benbp

Comment (m-nash) [reply sdk/keyvault/tests.yml:7]: No thank you, @ArthurMa1978 can work that out when they move to a process where the live tests are managed / monitored.

**Pull request merged**